src/hotspot/share/jvmci/jvmci_globals.cpp

```diff
@@ -86,6 +86,9 @@ bool JVMCIGlobals::check_jvmci_flags_are_consistent() {
       return false;
     }
     FLAG_SET_DEFAULT(EnableJVMCI, true);
+    if (FLAG_IS_DEFAULT(EagerJVMCI) && !EagerJVMCI) {
+      FLAG_SET_DEFAULT(EagerJVMCI, true);
+    }
```

Comment:
The default value is `false` - I don't think you need check it.
You can use `FLAG_SET_ERGO_IF_DEFAULT(EagerJVMCI, true);`